pkgs/tools/wayland/hyprland-iio/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, stdenv, fetchFromGitHub, meson, cmake, pkg-config, dbus, ninja, ... }:
```

Comment:
Remove the `...` and optionally use [`nixfmt-rfc-style`](https://search.nixos.org/packages?channel=unstable&show=nixfmt-rfc-style) to automatically format this file